[2025-06-15T13:35:44.689+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-15T13:35:44.758+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_etl_to_postgres.fetch_pbp_and_load_to_postgres manual__2025-06-15T13:35:41.889952+00:00 [queued]>
[2025-06-15T13:35:44.791+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_etl_to_postgres.fetch_pbp_and_load_to_postgres manual__2025-06-15T13:35:41.889952+00:00 [queued]>
[2025-06-15T13:35:44.793+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-15T13:35:44.873+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_pbp_and_load_to_postgres> on 2025-06-15 13:35:41.889952+00:00
[2025-06-15T13:35:44.890+0000] {standard_task_runner.py:63} INFO - Started process 810 to run task
[2025-06-15T13:35:44.899+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_etl_to_postgres', 'fetch_pbp_and_load_to_postgres', 'manual__2025-06-15T13:35:41.889952+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp06v3g85z']
[2025-06-15T13:35:44.902+0000] {standard_task_runner.py:91} INFO - Job 20: Subtask fetch_pbp_and_load_to_postgres
[2025-06-15T13:35:45.029+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_etl_to_postgres.fetch_pbp_and_load_to_postgres manual__2025-06-15T13:35:41.889952+00:00 [running]> on host e9ac9ffe8f89
[2025-06-15T13:35:45.244+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nba_etl_to_postgres' AIRFLOW_CTX_TASK_ID='fetch_pbp_and_load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-06-15T13:35:41.889952+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-15T13:35:41.889952+00:00'
[2025-06-15T13:35:45.246+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-15T13:35:45.592+0000] {logging_mixin.py:188} INFO - Retrying after 3528 sec...
[2025-06-15T13:36:45.768+0000] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to skipped. Terminating instance.
[2025-06-15T13:36:45.771+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-06-15T13:36:45.774+0000] {process_utils.py:132} INFO - Sending 15 to group 810. PIDs of all processes in the group: [810]
[2025-06-15T13:36:45.774+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 810
[2025-06-15T13:36:45.775+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-06-15T13:36:45.776+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-15T13:36:45.868+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=810, status='terminated', exitcode=0, started='13:35:44') (810) terminated with exit code 0
